src/Tree/HydroTree.cpp

```diff
@@ -191,6 +203,23 @@ TreeBase<ndim>* HydroTree<ndim,ParticleType>::CreateTree
 }
 
 
+template <int ndim, template <int> class ParticleType>
+bool HydroTree<ndim,ParticleType>::GetRelativeOpeningCriterion() const {
+  return tree->GetRelativeOpeningCriterion() ;
+}
+
+template <int ndim, template <int> class ParticleType>
+void HydroTree<ndim,ParticleType>::SetRelativeOpeningCriterion(bool value) {
+  tree->SetRelativeOpeningCriterion(value) ;
+  ghosttree->SetRelativeOpeningCriterion(value) ;
+#ifdef MPI_PARALLEL
+  for (int i=0; i<Nmpi; i++) {
+    prunedtree[i]->SetRelativeOpeningCriterion(value) ;
+    sendprunedtree[i]->SetRelativeOpeningCriterion(value) ;
```

Comment:
I don't think there is any need to set the relative opening criterion on the ghost tree as it is never used for gravity. Having said that, it's probably better to do it anyway for extra safety.